src/dask_awkward/lib/core.py

```diff
@@ -2051,3 +2059,13 @@ def typetracer_from_form(form: Form) -> ak.Array:
     """
     layout = form.length_zero_array(highlevel=False)
     return ak.Array(layout.to_typetracer(forget_length=True))
+
+
+def make_unknown_length(array: ak.Array) -> ak.Array:
+    return ak.Array(
+        ak.to_backend(
+            array,
+            "typetracer",
+            highlevel=False,
+        ).to_typetracer(forget_length=True)
+    )
```

Comment:
This can just be `to_layout(array).to_typetracer(...)` :)

Reply:
:+1: